src/store/actions/keymap.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@angular/core';
+import { Action } from '@ngrx/store';
+
+import { Keymap } from '../../config-serializer/config-items/Keymap';
+
+@Injectable()
+export class KeymapActions {
+    static PREFIX = '[Keymap] ';
```

Comment:
I know, static is not possible in class. I think that we don't need to implement anything special for this scenario, but if it's absolutely necessary let me know and will implement it.